rlkit/launchers/launcher_util.py

```diff
@@ -150,6 +150,7 @@ def run_experiment_here(
     experiment_uuid = actual_log_dir.split("/")[-1]
     run = None
     if variant.get("wandb", False):
+        logger.use_wandb = True
```

Comment:
why is this hardcoded true here, if wandb is false

Reply:
isn't wandb always true when we execute logger.use_wandb? like doesn't variant.get("wandb", false) autopopulate with false if the key doesn't exist, but if it does exist and is true, then we enter the conditional?

Reply:
oh my bad, I misread haha, yeah this is fine